dashboards-observability/public/components/explorer/hooks/use_fetch_events.ts

```diff
@@ -126,6 +132,29 @@ export const useFetchEvents = ({
     });
   };
 
+  const getLiveTail = (query: string = '', errorHandler?: (error: any) => void) => {
+    const cur = queriesRef.current;
+    const searchQuery = isEmpty(query) ? cur![requestParams.tabId][FINAL_QUERY] : query;
+    fetchEvents({ query: searchQuery }, 'jdbc', (res: any) => {
+      if (!isEmpty(res.jsonData)) {
+        if (!isEmpty(responseRef.current)){
+          res.jsonData = res.jsonData.concat(responseRef.current.jsonData);
+          res.datarows = res.datarows.concat(responseRef.current.datarows);
+          res.total = res.total + responseRef.current.total;
+          res.size = res.size + responseRef.current.size;
+        }
+        // send only first 200 data log arrays
+        if (res.total > 200) {
+          res = res.slice(0,200);
+        }
```

Comment:
If we are keeping only 200 items from total result, do we know are they latest 200 or the oldest 200. For example for an application log index, user would prefer latest 200 in reverse chronological order.

Reply:
you may check if the result is already in a sorted fashion, if not you can add "sort" command in the requested ppl query.

Reply:
The total limit for the logs to be displayed in the explorer page is 200 so i prepended the latest data to the existing. so i sliced the first 200 data logs. it'll cut off the old data at the end.

Reply:
Yeah okay, i'll do that